crates/protogen/proto/metastore/options.proto

```diff
@@ -204,10 +208,9 @@ message TableOptionsMongo {
   string connection_string = 1;
   string database = 2;
   string collection = 3;
-  repeated InternalColumnDefinition columns = 4;
```

Comment:
Is this intentional?

Reply:
Yes, i thought we agreed that the columns would only be applied to the `TableEntry`, not the `TableOptions`.